react/components/Profile/NewsletterBox.tsx

```diff
@@ -8,6 +8,7 @@ import ContentBox from '../shared/ContentBox'
 import GET_NEWSLETTER from '../../graphql/getNewsletterOpt.gql'
 import NEWSLETTER_MUTATION from '../../graphql/setOptInNewsletter.gql'
 import styles from '../../styles.css'
+import className from "../../styles/ContentBox.module.css";
```

Comment:
Since this is the NewsletterBox I would change it to a different file
```suggestion
import className from "../../styles/NewsletterBox.css";
```